data/blog/2024/2024-07-23-current-maintenance-status-of-laminas-mezzio-packages.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 2024-07-23-current-maintenance-status-of-laminas-mezzio-packages
+author: julian
+title: 'Current Maintenance Status of Laminas & Mezzio Packages'
+draft: false
+public: true
+created: '2024-07-23T11:00:00-01:00'
+updated: '2024-07-23T11:00:00-01:00'
+tags:
+  - laminas api tools
+  - custom proprieties
+  - maintenance status
+---
+
+Laminas Project has created a large number of packages to serve the needs of the PHP community. Unfortunately, there is
+no aggregated place to see the status of each package easily. You would need to visit each one of them on GitHub and
+check out the *Custom Properties* page. And that is when you remember to do so...
```

Comment:
In each repository, the page is called 'Custom Proprieties" page, that why I used that name 
The name of the class will be changed in the next PR 
![image](https://github.com/user-attachments/assets/fe7d824d-0af8-4fdf-a739-5b42a7df7a1f)

Reply:
Right, the page on GitHub is named that way, but we do not want to push the user in that direction.
As a reminder:

1. Creating the overview page
1. Adding the badge to each component documentation
1. Updating the Readme file

The typical user will not look at the custom properties page, and with these steps it is not necessary.

Reply:
To clarify my suggestion: put the description of the badges at the end of the overview page and rewrite the blog post so that it is an advertisement for the new overview. This means that all important information should be contained on the maintaince overview page and not one part in the blog and the other part in the overview.